gisMetadataProcessing.py

```diff
@@ -0,0 +1,81 @@
+from bs4 import BeautifulSoup
+import csv
+import os
+
+
+def creatkeyword(value):
+    """Create keyword element."""
+    keywordkey = keywordxml.new_tag(tag + 'key')
+    keywordkey.string = value
+    if vocab == 'fst':
+        getfasturi(value, keywordkey)
+    else:
+        pass
+    keyword.append(keywordkey)
+
+
+def getfasturi(value, keywordkey):
+    """Get URI for FAST term."""
+    urn = ''
+    fastcsv = csv.DictReader(open('fast.csv'))
+    for row in fastcsv:
+        if value == row['label']:
+            urn = row['uri']
+            print(urn)
+            break
+        else:
+            pass
+    if urn == '':
+        print('No FAST URI - check value in spreadsheet')
+        exit()
+    else:
+        keywordkey['urn'] = urn
+
+
+def splitkeywords(fieldname):
+    """Split multiple keyword values separated by pipe delimiters."""
+    values = row[fieldname].split('|')
+    for value in values:
+        creatkeyword(value)
+
+
+dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I would generally avoid overwriting any of the built-in functions at the global level. If someone were to open up a Python REPL and do `from gisMetadataProcessing import *` they would find that `dir()` no longer works as expected. OK to use `dir` as a variable name in local scope, though.

Reply:
Copied this from a Stack Overflow answer and forgot that dir is a function, I'll rename this